CHANGELOG.md

```diff
@@ -60,6 +72,7 @@ Note, since the pipeline is using Nextflow DSL2, each process will be run with i
 | fastk           |             | f18a4e6d2207539f7b84461daebc54530a9559b0 |
 | freebayes       |             | 1.3.6                                    |
 | gatk4           |             | 4.4.0.0                                  |
+| genescope       |             | 380815c420f50171f9234a0fd1ff426b39829b91 |
```

Comment:
Is this the new dependency for the new release? Then it should not be here.

Reply:
It's the old dependency missed in this list in the previous released.